OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -68,6 +68,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Is the missile blocked by actors with BlocksProjectiles: trait.")]
 		public readonly bool Blockable = true;
 
+		[Desc("Ignore actor, who fired the projectile, when detecting blocking actors?")]
+		public readonly bool BlockingIgnoreSourceActor = false;
```

Comment:
```suggestion
		public readonly bool IgnoreBlockingSourceActor = false;
```

Reply:
Like @PunkPun said, he wants to have some of the features, we wanted to have in OpenE2140 (by using changes in this PR), in the OpenRA itself. So I think this PR is currently not in state for merging/reviewing.